modules/xcode/xcode_common.lua

```diff
@@ -959,6 +959,40 @@
 	end
 
 
+	function xcode.XCBuildConfiguration_CLanguageStandard(settings, cfg)
+		if (p.languages.isc(cfg.language)) then
+			if cfg.language ~= "C" then
+				settings['GCC_C_LANGUAGE_STANDARD'] = cfg.language:lower()
+			else
+				if cfg.flags['C99'] then
+					settings['GCC_C_LANGUAGE_STANDARD'] = 'C99'
+				elseif cfg.flags['C11'] then
+					settings['GCC_C_LANGUAGE_STANDARD'] = 'C11'
+				else
+					settings['GCC_C_LANGUAGE_STANDARD'] = 'C90'
+				end
+			end
+		else
+			settings['GCC_C_LANGUAGE_STANDARD'] = 'gnu99'
```

Comment:
Should this map to one of the C11 standards when the language is C++17? I believe C++17 is based on C11, rather than C99, or is it the plan for the next C++ standard to be on C11?